pandas/io/ga.py

```diff
@@ -8,16 +8,12 @@
 import re
 from pandas import compat
 import numpy as np
-from pandas import DataFrame
 import pandas as pd
 import pandas.io.parsers as psr
-import pandas.lib as lib
-from pandas.io.date_converters import generic_parser
 import pandas.io.auth as auth
 from pandas.util.decorators import Appender, Substitution
 
 from apiclient.errors import HttpError
-from oauth2client.client import AccessTokenRefreshError
```

Comment:
no idea if this is needed.

to be honest I would prob ``#no qa`` the entire module as we deprecated this in 0.17.1 (and intention to blow away soon)